llvm/test/CodeGen/AArch64/sme2p1-intrinsics-zero.ll

```diff
@@ -0,0 +1,94 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 4
+; RUN: llc -verify-machineinstrs < %s | FileCheck %s
+
+target triple = "aarch64-linux"
+
+define  void @test_svzero_za64_vg1x2(i32  %slice)  #0 {
```

Comment:
I have added the test, but offset creation is not working as expected. I can try to debug why as part of this patch, but maybe it would be better to do it as separate pull request as I do not think it is unique to this intrinsic. What do you think ?

Reply:
Done